README.md

```diff
@@ -61,6 +61,10 @@ please join us in
 
 This library should always compile with any combination of features on **Rust 1.29**.
 
+```
+rustup default 1.29.0
```

Comment:
Not sure whether its good to ask user to change the default rust version - especially with the fact that it will compile with any version above that

Reply:
I added this to help a contributor test with MSRV. Maybe add a note saying "To test with MSRV..." ?

Reply:
For testing I would prefer something like `rustup install 1.29.0 && cargo +1.29.0 build`, which will not switch the global rust environment

Reply:
Right, just `default` subcommand also  assumes 1.29.0 is already installed.